server/filestore.go

```diff
@@ -6445,6 +6445,14 @@ func (o *consumerFileStore) Type() StorageType { return FileStorage }
 // State retrieves the state from the state file.
 // This is not expected to be called in high performance code, only on startup.
 func (o *consumerFileStore) State() (*ConsumerState, error) {
+	return o.stateWithCopy(true)
+}
+
+func (o *consumerFileStore) BorrowState() (*ConsumerState, error) {
```

Comment:
What am I missing? I don't see this function being called?

Reply:
lol forgot to plug in. fixed.